success_stories/styles.css

```
@@ -139,9 +142,10 @@ hr {
     margin-bottom: -1px;
 }
 
-.mentor-details .details{
+.mentor-details .details {
     line-height: 1.2rem;
-    margin-top: 4px;;
+    margin-top: 4px;
+    ;
```

Remove the extra semicolon
```suggestion
    margin-top: 4px;
```